### DISCUSSION TITLE
Using high-contrast skin, but menu toggle button flagged as low contrast by accessibility checking software

### DISCUSSION THREAD

**Discussion by ladylazarus3** [mmistakes/minimal-mistakes#3913] [General]
Hi folks,  

This is my first post, but i've been an active user of Minimal Mistakes since 2020, and it has saved my bacon so many times.  What a wonderful community.  Thank you to Michael, and to all contributors, that make this such a fantastic resource, the fabulous documentation, and excellence in maintaining it over the long-term.

i've been doing a bit of web accessibility testing (mostly with the WAVE Web Accessibility Evaluation Tool available at https://wave.webaim.org/ ) and even though i'm using the "High Contrast" skin for Minimal Mistakes, the web accessibility tool keeps flagging the color of the "Toggle menu +" button (i.e., the button that opens the left-hand navigational menu when viewed on a small screen/mobile device)  as "Very low contrast between text and background colors."  i can probably poke around in the css to change this, but i'm wondering if others are seeing the same results and a configuration fix could be added to the next Minimal Mistakes release to address the accessibility issue broadly for other Minimal Mistakes users as well.

Thank you again for this community and resource.  i hope this request made sense, and i really appreciate everyone's time in considering this proposal.

k

**Comment by iBug** [comment]
Online tools have been known to give false positives on all kinds of reports. I just checked [mine](https://ibug.io/blog) and it told me "very low contrast" for the white text on blue background in the footer area.

My recommendation would be to take these automated checks with a grain of salt, and double-check the reports by yourself. Most often these are really non-issues.

**Reply by ladylazarus3** [reply]
@iBug Thank you so much for your very kind response.  <3  i really appreciate your time and expertise in web accessibility.  Certainly, this makes sense to me, and i up until now have had very little practical experience in web design.  If you would indulge me further, is there a sense of a WCAG rating for an "out of the box" Minimal Mistakes implementation?  Realizing, of course, that folks may have make significant theme configuration changes in their implementation.  Thank you again for your rapid response, and being willing to indulge me further.

k
